src/ralph/settings-test-ralph.py

```diff
@@ -18,3 +18,5 @@
 SOUTH_TESTS_MIGRATE = False
 
 ASSETS_AUTO_ASSIGN_HOSTNAME = True
+
+INSTALLED_APPS += ['ralph.util.tests']
```

Comment:
It's probably added here to use `MyDevice` model in unit tests. Please look at Scrooge solution: https://github.com/allegro/ralph_pricing/blob/develop/src/ralph_scrooge/tests/models.py#L22 - maybe adding `Meta` with `app_label` will be sufficient here (instead of adding `ralph.util.tests` to `INSTALLED_APPS`)